src/test/mon/test-mon-msg.cc

```diff
@@ -308,6 +310,35 @@ TEST_F(MonMsgTest, MRouteTest)
   ASSERT_EQ(PTR_ERR(r), -ETIMEDOUT);
 }
 
+TEST_F(MonMsgTest, MMonCommandTest)
```

Comment:
@rzarzynski - spoke to @leonid-s-usov regarding this and this test doesn't seem to run from anywhere. Is it a leftover? Would you know?